protocol/app/app.go

```diff
@@ -1418,14 +1396,21 @@ func (app *App) initOracle(appOpts servertypes.AppOptions) {
 		panic(err)
 	}
 	// Create the oracle service.
-	app.oracleClient, err = oracleclient.NewClientFromConfig(
+	slinkyClient, err := oracleclient.NewClientFromConfig(
 		cfg,
 		app.Logger().With("client", "oracle"),
 		oracleMetrics,
 	)
 	if err != nil {
 		panic(err)
 	}
+
+	app.oracleClient = vote_extensions.NewOracleClient(
+		slinkyClient,
+		app.PricesKeeper,
+		&daemontypes.GrpcClientImpl{},
```

Comment:
Do we need the PriceFeedClient here to update the indexPriceCache? I was thinking that we could j pass the indexPriceCache obj. here